### PR TITLE
Add template for pages with option to add blocks

### PR DESCRIPTION
**Pull request by pigeonfresh**
With this additional template an user can create a page and when it uses the wizard it can add blocks to the page.

### CODE REVIEW

File: template/page/{name_sc}.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"title": "{{name_pc}}"{{#if blocks}},
+	"blocks": [{{#each blocks}}
+		{
+			"name" : "{{this}}"
```
Comment (ThaNarie) [inline template/page/{name_sc}.json]: I would also add an empty data object, so it's obvious where to place stuff

File: yarn.lock [hunk 2]
```diff
@@ -19,21 +19,34 @@ acorn-dynamic-import@^2.0.0:
   dependencies:
```
Comment (ThaNarie) [inline yarn.lock]: Why is this file in this diff? Do you have the latest yarn installed?

Comment (pigeonfresh) [inline yarn.lock]: Like discussed I probably had an newer version of yarn and upgraded to the latest version

**Comment by pigeonfresh** [comment]
I've added an empty data object to the page template as requested

File: script/template-standalone.hbs [hunk 3]
```diff
@@ -8,9 +8,9 @@
 
 	<meta httpEquiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
 	<meta name="viewport" content="width=device-width, user-scalable=no" />
-	<link rel="icon" href="/favicon.ico" type="image/x-icon" />
-	<link rel="stylesheet" href="../asset/common.css">
-	<link rel="stylesheet" href="../asset/{{page}}.css">
+	<link rel="icon" href="favicon.ico" type="image/x-icon" />
+	<link rel="stylesheet" href="asset/common.css">
+	<link rel="stylesheet" href="asset/bundle.css">
```
Comment (ThaNarie) [inline script/template-standalone.hbs]: Should not be done!

**Pull request closed without merging**